media/common/src/main/java/io/helidon/media/common/ContentWriters.java

```diff
@@ -81,6 +83,44 @@ private ContentWriters() {
         return Single.just(DataChunk.create(false, buffer.encode(charset)));
     }
 
+    /**
+     * Create a a publisher {@link DataChunk} with the given
+     * {@link Throwable} / {@link Charset} and return a {@link Single}.
+     *
+     * @param throwable the {@link Throwable}
+     * @param context a {@link MessageBodyWriterContext}
+     * @param setContentLength whether {@link MessageBodyWriterContext#contentLength(long)} should be called
+     * @return Single
+     */
+    public static Single<DataChunk> writeStackTrace(Throwable throwable,
+                                                    MessageBodyWriterContext context,
```

Comment:
With the addition the context as parameter, this methods differs from all the other ones in `ContentWriters`.
Lets not over do it and stick with just `ThrowableBodyWriter`.

Reply:
OK; note that the content length issue is one that will be faced by the `CharSequence`-related methods as well, so if it's OK to remove content length concerns then I can accept what they do.  WDYT?